elasticdl/pkg/commonnew/tensor.go

```diff
@@ -168,3 +169,12 @@ func (t *Tensor) SerializeTensor() *tensor_go_proto.TensorProto {
 		Dtype:         t.Dtype,
 	}
 }
+
+// SerializeToIndexedSlices return proto.IndexedSlices
+func (t *Tensor) SerializeToIndexedSlices(indices []int64) *proto.IndexedSlices {
```

Comment:
Need to check the Tensor dim and indices length here. If the dims do not match, then return a `nil` pointer.

Reply:
fixed

Reply:
It seems that it's still not fixed yet.

Reply:
fixed now...

Reply:
Sorry, I do not express it clearly.
It is saied to check the dim of Tensor and indices. The tensor must be a 2-dim tensor, and the row dim of the tensor  is equal to `len(indices)`.